test/contract/arbRollup.spec.ts

```diff
@@ -789,6 +802,44 @@ describe('ArbRollup', () => {
     await rollup.confirmNextNode(challengerNode)
   })
 
+  it('allow force refund staker with pending node', async function () {
+    await rollupAdmin.connect(await impersonateAccount(upgradeExecutor)).pause()
```

Comment:
missing .wait() - and on some other transactions below

Reply:
that should not be necessary right? otherwise the test should be failing

Reply:
could just be getting lucky with timing. You might get intermittent failures with this